manager/base.py

```diff
@@ -665,15 +667,35 @@ def parse_tags(tags_list):
     return tags
 
 
+def inventory_groups_condition(group_ids) -> Tuple[str, bool]:
```

Comment:
please make sure the filtering with where works correctly,
this function return the following output for given inputs
```python
In [26]: x = [{"id": "group1"}, {"id": "group2"}]

In [27]: inventory_groups_condition(x)
Out[27]: ('{"{\\"id\\": \\"group1\\"}","{\\"id\\": \\"group2\\"}"}', False)
```
I'm not sure if this is the correct format for ANY @> filter, I think it should be `'{"[{\\"id\\": \\"group1\\"}]","[{\\"id\\": \\"group2\\"}]"}'`

```python
In [28]: inventory_groups_condition([[]])
Out[28]: ('{}', True)
```
if there are only ungrouped system groups (only empty values - `[]`), then we don't want to use where clause at all, so you might want to return `('{}', False)` from this function